tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py

```diff
@@ -1010,6 +1012,36 @@ def RunTest(self, run_params):
           atol=self.ExpectedAbsoluteTolerance(run_params),
           rtol=self.ExpectedRelativeTolerance(run_params))
 
+  @contextmanager
+  def _experimental_feature_scope(self, feature_name):
+    """Test ContextManager enabling an experimental feature.
```

Comment:
How about this single line summary:
Creates a context manager to enable the given experimental feature.

Reply:
Fixed